lodash/lodash.d.ts

```diff
@@ -4559,6 +4559,18 @@ declare module _ {
         sample<T>(collection: Dictionary<T>, n: number): T[];
     }
 
+    interface LoDashArrayWrapper<T> {
+        /**
+         * @see _.sample
+         **/
+        sample(n: number): LoDashArrayWrapper<T>;
+
+        /**
+         * @see _.sample
+         **/
+        sample(): LoDashArrayWrapper<T>;
```

Comment:
If you return just `T`, then you can no longer continue the `_.chain`. Am I right?